spec/ameba/config_spec.cr

```diff
@@ -84,6 +84,12 @@ module Ameba
         config.formatter.should_not be_nil
       end
 
+      it "raises when custom config file doesn't exist" do
+        expect_raises(Exception, "Config file is invalid: Unable to find config file foo.yml") do
+          Config.load "foo.yml"
+        end
```

Comment:
can we rephrase the error message re-thrown from `load` method now? Maybe something like

`Config file is invalid` -> `Unable to load config file`

Previously the only reason why it was thrown is invalid YAML. Now the file may not exist here.

Reply:
sure thing good shout :) 4741c9f